Readme.md

```diff
@@ -484,7 +439,41 @@ class OrganizationFinder extends AbstractDoctrineFinder
 }
 ```
 
-... todo: more explanations
+### How to implement you own event listener ?
+
+To implement other custom event listeners you only need to implement the `EventListenerInterface`:
+
+```php
+class ConsoleOutputListener implements EventListenerInterface
+{
+    /**
+     * @var SystemLoggerInterface
+     * @Flow\Inject
+     */
+    protected $systemLogger;
+
+    /**
+     * @param ButtonTagged $event
+     * @param EventMetadata $metadata
+     */
+    public function whenButtonTagged(ButtonTagged $event, RawEvent $rawEventData)
+    {
+        $this->systemLogger->log('--- ConsoleOutputListener say something has been tagged ---');
+    }
+}
+```
+
+The event handler locator can throw an exception if something is wrong with your command handler definition. In that
+case please check your system log for more information.
+
+All the wiring between event and event listeners is done automatically, if you respect the following convention:
+
+* Method name must be when[ShortEventName]
+* The first parameter must be of type `EventInterface` and is your concrete DomainEvent instance
+* The second parameter is optional and is of type `RawEvent`, containing the raw data and metadata from the EventStore
+
+Also, you can optionally let your `EventListener` implement `ActsBeforeInvokingEventListenerMethodsInterface`
+in order to receive a hook method `beforeInvokingEventListenerMethod` before the concrete event handling method (`when*`) is called.
```

Comment:
Actually, I'm not so sure anymore what this was done for. Originally, it was an interface to manually override the handler method invocation, but changed later on. Maybe @robertlemke can give an usage example?